### PR TITLE
docs: include `envVar`

### LINKED ISSUE #2449: feat: `--env-var` for library usage
1. Newman Version (can be found via `newman -v`): 5.1.2
 2. OS details (type, version, and architecture): N/A
 3. Are you using Newman as a library, or via the CLI? Library

---

I'd like to see a function that sets environment variables while using the `environment` key in `options`. This is good for secrets and other things.

### PR DESCRIPTION
**Pull request by jef**
### Description

Closes #2449

### CODE REVIEW

**Comment by DannyDainton** [comment]
@jef Did you also want to extend this out to cover the `globalVar` option?

**Comment by jef** [comment]
I'll also include some examples via code.

**Comment by jef** [comment]
Okay, looks good on my end.

**Comment by jef** [comment]
Feel free to merge if you think it's good to go now!

Thanks for the review.

**Review by codenirvana — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -356,6 +360,10 @@ newman.run({
         "_postman_exported_at": "2016-10-17T14:26:34.940Z",
         "_postman_exported_using": "Postman/4.8.0"
     }
+    envVar: [
```
Comment (disenhower) [inline README.md:363]: This was the only place I could find an example of how to use envVar.  It would be nice to get this included in the documentation.

Comment (jef) [reply README.md:363]: This merged PR is for the documentation...

image

**Pull request merged**